pages/form-builder/contactus.tsx

```diff
@@ -0,0 +1,281 @@
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { GetServerSideProps } from "next";
+
+import React, { useState } from "react";
+import Link from "next/link";
+import { Formik } from "formik";
+import { useTranslation } from "next-i18next";
+import * as Yup from "yup";
+import axios from "axios";
+import { logMessage } from "@lib/logger";
+import {
+  Button,
+  TextInput,
+  Label,
+  Alert,
+  ErrorListItem,
+  MultipleChoiceGroup,
+  TextArea,
+  Description,
+} from "@components/forms";
+import { StyledLink } from "@components/globals/StyledLink/StyledLink";
+
+export default function Contactus() {
+  const { t, i18n } = useTranslation(["form-builder", "common"]);
+  const [errorState, setErrorState] = useState({ message: "" });
+  const [submitting, setSubmitting] = useState(false);
+  const [isSuccessScreen, setIsSuccessScreen] = useState(false);
+
+  const handleRequest = async (
+    name: string,
+    email: string,
+    request: string,
+    description: string
+  ) => {
+    const type = "contactus";
+    return await axios({
+      url: "/api/request/help",
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      data: { type, name, email, request, description },
+      timeout: process.env.NODE_ENV === "production" ? 60000 : 0,
+    }).catch((err) => {
+      logMessage.error(err);
+      setErrorState({ message: t("submissionError") });
+    });
+  };
+
+  const validationSchema = Yup.object().shape({
+    name: Yup.string()
+      .required(t("input-validation.required", { ns: "common" }))
+      .max(50, t("signUpRegistration.fields.name.error.maxLength")),
```

Comment:
Probably not a good idea here to use the same translation as the name field in the signUp scope. Yes its repetition but it ensures we know what translation strings affect the different parts of the app. Also I don't see this namespace in the `useTranslation` so this probably would not work

Reply:
Oh crap I didn't even notice that, yup totally agree :)

Reply:
There is also no max length in the design so I'll remove it